packages/gatsby-source-filesystem/src/__tests__/utils.js

```diff
@@ -22,3 +22,13 @@ describe(`create remote file node`, () => {
     })
   })
 })
+
+describe(`slash path`, () => {
+  it(`can correctly slash path`, () => {
+    ;[[`foo\\bar`, `foo/bar`], [`foo\\中文`, `foo/中文`]].forEach(
+      ([path, expectRes]) => {
+        expect(slash(path)).toBe(expectRes)
+      }
+    )
+  })
+})
```

Comment:
please add a second test asserting that extended length paths are not modified

Reply:
thanks for review,I have already updated.

Reply:
Athang663